fedbiomed/common/message.py

```diff
@@ -83,10 +85,24 @@ def __post_init__(self):
         """
 
         if not self.__validate(self.__dataclass_fields__.items()):
-            _msg = ErrorNumbers.FB601.value + ": bad input value for message: " + self.__str__()[0:1000] + "..."
+            _msg = (
+                ErrorNumbers.FB601.value
+                + ": bad input value for message: "
+                + self.__str__()[0:1000]
+                + "..."
+            )
             logger.critical(_msg)
             raise FedbiomedMessageError(_msg)
 
+    @property
+    def __name__(self) -> str:
```

Comment:
Thank you @srcansiz for this nice refactor !

I would propose to use `name()` rather than `__name__()` as it is meant to be a public method.

```suggestion
    def name(self) -> str:
```

Reply:
Hi @mvesin,

`__name__` is known convention to get class name, and using it outside of the class is not against PEP8 or flake8. Therefore, I wanted to use `__name__`, and it is a property rather than a method. So it is used as `message.__name__`. I can not do the same `message.name` because `name` can be a property of a dataclass.